### ISSUE TITLE
Exporting to PDF with data-tableexport-value

### ISSUE THREAD

**Issue by jdlv1** [hhurz/tableExport.jquery.plugin#297]
Hi and thanks for the great plugin.  There seems to be a bug when explicitly stating "data-tableexport-value" and exporting to PDF.  The "data-tableexport-value" does not export.  Exporting to MS Excel works as expected.

```
<script src="//cdn.jsdelivr.net/npm/tableexport.jquery.plugin@1.10.17/tableExport.min.js"></script>
<script src="/_script/filesaver.min.js"></script>
<script src="//unpkg.com/tableexport.jquery.plugin/libs/jsPDF/jspdf.min.js"></script>
<script src="//unpkg.com/tableexport.jquery.plugin/libs/jsPDF-AutoTable/jspdf.plugin.autotable.js"></script>
<script src="//unpkg.com/bootstrap-table@1.16.0/dist/bootstrap-table.min.js"></script>
<script src="//unpkg.com/bootstrap-table@1.16.0/dist/extensions/export/bootstrap-table-export.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/xlsx/0.15.6/xlsx.core.min.js"></script>

<table id="table-results" class="table"
data-show-export="true"
data-pagination="true"
data-side-pagination="client"
data-click-to-select="true"
data-show-toggle="true"
data-show-columns="true"
data-page-size="50"
data-page-list="[50, 100, 300, 500, all]"
data-export-data-type="all">

<thead>
<tr>
<th>First name</th>
<th>Last name</th>
<th>Group</th>
</tr>
</thead>

<tbody>
<tr>
<td>Jane</td>
<td>Doe</td>
<td data-tableexport-value="Group 1, Group 2, Group 3">Group 1 +2</td>
</tr>
</tbody>
</table>

<script>
var $table = $('#table-results');
$(function() {
  $table.bootstrapTable('destroy').bootstrapTable({
	exportDataType: 'all',
	exportTypes: ['json', 'xml', 'csv', 'txt', 'sql', 'excel', 'xlsx', 'pdf'],
	exportOptions: {
		fileName: 'user information report', //file name setting
		excelstyles: ['background-color', 'color', 'font-size', 'font-weight'],
		jspdf: {
			format: 'bestfit',
			autotable: {
				theme: 'grid',
				bodyStyles: {
					lineColor: 128
				},
				styles: {
					overflow: 'linebreak',
					fontSize: 8
				},
				alternateRowStyles: {
					fillColor: [255, 255, 255]
				}
			}
		}
	}
  });
</script>

```

What did I miss?  Seems like the only workaround would be hiding an alternate column and forcing cells to export with data-tableexport-display="always"?

**Comment by hhurz** [comment]
Hi, after I got your example code running, everything works fine for me. I think there are some mismatches in your script includes. Here are your modified example and an expected pdf result:
``` html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Issue297</title>
    <script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>

    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.6/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.16.0/bootstrap-table.min.js"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.16.0/extensions/export/bootstrap-table-export.js"></script>

    <script src="//cdn.jsdelivr.net/npm/tableexport.jquery.plugin@1.10.17/tableExport.min.js"></script>
    <script src="//unpkg.com/tableexport.jquery.plugin/libs/jsPDF/jspdf.min.js"></script>
    <script src="//unpkg.com/tableexport.jquery.plugin/libs/jsPDF-AutoTable/jspdf.plugin.autotable.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/xlsx/0.15.6/xlsx.core.min.js"></script>

    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" />
    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.10.1/bootstrap-table.min.css" />

    <script>
      $(document).ready(function()
      {
        var $table = $('#table-results');
          $table.bootstrapTable('destroy').bootstrapTable({
            exportDataType: 'all',
            exportTypes: ['json', 'xml', 'csv', 'txt', 'sql', 'excel', 'xlsx', 'pdf'],
            exportOptions: {
              fileName: 'user information report', //file name setting
              excelstyles: ['background-color', 'color', 'font-size', 'font-weight'],
              jspdf: {
                format: 'bestfit',
                autotable: {
                  theme: 'grid',
                  bodyStyles: {
                    lineColor: 128
                  },
                  styles: {
                    overflow: 'linebreak',
                    fontSize: 8
                  },
                  alternateRowStyles: {
                    fillColor: [255, 255, 255]
                  }
                }
              }
            }
          });
      });
    </script>
</head>
<body>
    <div class="container">
        <table id="table-results" class="table"
               data-show-export="true"
               data-pagination="true"
               data-side-pagination="client"
               data-click-to-select="true"
               data-show-toggle="true"
               data-show-columns="true"
               data-page-size="50"
               data-page-list="[50, 100, 300, 500, all]"
               data-export-data-type="all">

            <thead>
            <tr>
                <th>First name</th>
                <th>Last name</th>
                <th>Group</th>
            </tr>
            </thead>

            <tbody>
            <tr>
                <td>Jane</td>
                <td>Doe</td>
                <td data-tableexport-value="Group 1, Group 2, Group 3">Group 1 +2</td>
            </tr>
            </tbody>
        </table>
    </div>
</body>
</html>
```


[user information report.pdf](https://github.com/hhurz/tableExport.jquery.plugin/files/4548147/user.information.report.pdf)

**Comment by jdlv1** [comment]
Hi, you are totally right.  I tested again and it works correctly, thank you.

However, I failed to mention (**sorry**) that the cell is showing the count with an anchor link (href), like this:

````
<td data-tableexport-value="Group 1, Group 2, Group 3">Group 1 <a href="#" data-toggle="tooltip" title="Group 2, Group 3">+2</a></td>
````

As soon as the href is set, the "data-tableexport-value" seems to be ignored when exporting to PDF.

**Comment by hhurz** [comment]
Hi, I confirm this issue. It will be fixed in the next couple of days...

**Comment by hhurz** [comment]
Fixed in release 1.10.18. @jdlv1 Thanks for reporting this.

**Comment by jdlv1** [comment]
Works great! Thank you @hhurz!
